### DISCUSSION TITLE
[Bug?] No export project function

### DISCUSSION THREAD

**Discussion by jdnx** [NickeManarin/ScreenToGif#1216] [Q&A] [answered]
When I load a video file and finish editing frames, I want to save this project for future use , I press SaveAs but only Gif / Default Gif  2 type options but no project and other options(according software demo pictures I see 5 save type options 

I posted my Screenshots here 

 ![demo](https://github.com/NickeManarin/ScreenToGif/assets/38124494/24809bb2-1887-4195-bd91-7b62c2352fee)

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Version 2.38.1 portable

It maybe not a bug but my wrong configuration?  Please help, Thanks

**Comment by NickeManarin** [comment]
The first select element lets you pick the file type.
Just select "Project" and you can export in that format.

The next version of the app will handle better and differentiate between a project and a lossy exported file.

**Comment by jdnx** [comment]
> The first select element lets you pick the file type. Just select "Project" and you can export in that format.
> 
> The next version of the app will handle better and differentiate between a project and a lossy exported file.

@NickeManarin

Thanks, but I do not quite understand what you said, you mean this version(previous version has more save type options?) can only export to a GIF format file but no project file itself?  If so, does that mean if I do not export to gif after finish editing frames, this project can only be saved as a temporary project, and it will be removed automatically , so if I want to save this project for a long period time, I can only edit this app's setting and set a long day period for auto clean option?

**Comment by jdnx** [comment]
I searched internet and find following app demo picture
![R-C](https://github.com/NickeManarin/ScreenToGif/assets/38124494/1700d7d5-f573-4618-ae25-c336db035d2b)
it has a "project" SaveAs type, so this is a previous version? It can save project to a project file for future use without being cleaned automatically?

**Comment by NickeManarin** [comment, marked as answer]
No, you can still export as project as I wrote in my previous reply.
You just have to select the "Project" file type in the select/combobox element:

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/b2be2f33-1516-4910-84fc-4ceb511aa506)

It's at the bottom of the list of items.

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/887b9a44-4156-4063-ad08-f7b7e5fa74d7)

I also explained that the next version of the app (v3) will treat projects and gifs the same way as Photoshop treats PSD vs Jpeg files for example. Meaning that you'll be able to hit Ctrl + S and save as STG project or click on Export to export to a lossy format (gif, apng, mp4, webm etc.).

**Reply by NickeManarin** [reply]
I migrate this ticket to Q&A as it's a non-issue. :)
